src/core/ext/filters/client_channel/resolver/dns/event_engine/service_config_helper.cc

```diff
@@ -57,8 +55,8 @@ struct ServiceConfigChoice {
   }
 };
 
-template <typename T>
-bool vector_contains(const std::vector<T> v, const T& value) {
+bool vector_contains(const std::vector<std::string> v,
```

Comment:
This parameter should be a reference to avoid an unnecessary copy.

Reply:
Good catch, fixed.